tfs/src/main/java/hudson/plugins/tfs/model/BuildCommand.java

```diff
@@ -209,23 +209,27 @@ static void contributeTeamBuildParameterActions(final Map<String, String> teamBu
             final boolean isTeamGit = "TfGit".equalsIgnoreCase(provider)
                     || "TfsGit".equalsIgnoreCase(provider);
             if (isTeamGit) {
-                final String collectionUriString = teamBuildParameters.get(SYSTEM_TEAM_FOUNDATION_COLLECTION_URI);
-                final URI collectionUri = URI.create(collectionUriString);
+            	// "Build.Repository.Uri" is null/whitespace/empty when the 'Jenkins Queue Job' task runs in TFS Release Management.
+            	// In this case, do not reference a CommitParameterAction in the actions reported as unsupported.
                 final String repoUriString = teamBuildParameters.get(BUILD_REPOSITORY_URI);
-                final URI repoUri = URI.create(repoUriString);
-                final String projectId = teamBuildParameters.get(SYSTEM_TEAM_PROJECT);
-                final String repoId = teamBuildParameters.get(BUILD_REPOSITORY_NAME);
-                final String commit = teamBuildParameters.get(BUILD_SOURCE_VERSION);
-                final String pushedBy = teamBuildParameters.get(BUILD_REQUESTED_FOR);
-                final GitCodePushedEventArgs args = new GitCodePushedEventArgs();
-                args.collectionUri = collectionUri;
-                args.repoUri = repoUri;
-                args.projectId = projectId;
-                args.repoId = repoId;
-                args.commit = commit;
-                args.pushedBy = pushedBy;
-                final CommitParameterAction action = new CommitParameterAction(args);
-                actions.add(action);
+                if (repoUriString != null && !repoUriString.trim().isEmpty()) {

```

Comment:
`Build.Repository.Uri` is present in the JSON payload and ends up having a value of white space.  So unfortunately, we can't use `teamBuildParameters.containsKey(BUILD_REPOSITORY_URI)`.  Great suggestion to use `org.apache.commons.lang.StringUtils.isNotBlank()`.  I made that change.  Does it look okay otherwise @olivierdagenais?